lib/runtime/version.go

```diff
@@ -30,9 +30,24 @@ type Version struct {
 }
 
 var (
+	// Blake2-8("TaggedTransactionQueue")
+	encodedTaggedTransactionQueue = [8]byte{0xd2, 0xbc, 0x98, 0x97, 0xee, 0xd0, 0x8f, 0x15}
+
 	ErrDecodingVersionField = errors.New("decoding version field")
 )
 
+// TaggedTransactionQueueVersion returns the TaggedTransactionQueueAPI version
+func (*Version) TaggedTransactionQueueVersion(runtimeVersion Version) uint32 {
```

Comment:
Why is this a method on Version and taking a Version argument? Remove the version argument and leave it as a method I'd say?

Reply:
Very good point. I initially built this when we supported the legacy version as well, so had receivers for both. Just forgot to remove